README.md

```diff
@@ -1,6 +1,26 @@
-# DevelopX
+# DevelopX-Expess API Usage
 
-``` npm install ``` <br/>
```

Comment:
both gif and commands should be here

Reply:
> its just suggestions. I think this code is pretty good. Now you need to add tests ;]

Did you check the test logs? 
- https://travis-ci.org/oraykt/DevelopX-Express

Reply:
https://github.com/oraykt/DevelopX-cli/commit/cf3b16b11abeed4d264e986cdccbc01d39669c05

> developx-cli@1.0.0 test /home/oraykt/Workspace/DevelopX-cli
> npm run test
>> developx-cli@1.0.0 mocha /home/oraykt/Workspace/DevelopX-cli
>> mocha "test-app.js"
>> node app.js
    ✓ Should logged Result (466ms)
    ✓ Should logged Error (456ms)

  2 passing (926ms) is available